cmd/query/app/server_test.go

```diff
@@ -517,7 +520,8 @@ func TestServerGRPCTLS(t *testing.T) {
 				},
 				GRPC: configgrpc.ServerConfig{
 					NetAddr: confignet.AddrConfig{
-						Endpoint: ":0",
+						Endpoint:  ":0",
+						Transport: confignet.TransportTypeTCP,
```

Comment:
is this now required? Is it not the default?

Reply:
It is the default inside the factory (see below). Previously, we were calling `net.Listen("tcp", s.queryOptions.GRPC.NetAddr.Endpoint)` which was hardcoding it when initializing the listener rather than using the transport set in the grpc config

```go
func createDefaultConfig() component.Config {
	return &Config{
		QueryOptions: app.QueryOptions{
			HTTP: confighttp.ServerConfig{
				Endpoint: ports.PortToHostPort(ports.QueryHTTP),
			},
			GRPC: configgrpc.ServerConfig{
				NetAddr: confignet.AddrConfig{
					Endpoint:  ports.PortToHostPort(ports.QueryGRPC),
					Transport: confignet.TransportTypeTCP,
				},
			},
		},
	}
}
```